platform/ios/src/MGLMapView.h

```diff
@@ -329,6 +329,13 @@ MGL_EXPORT
  */
 @property (nonatomic, assign) CGPoint scaleBarMargins;
 
+/**
+ The maximum width ratio of the scale bar relative to the map view's frame.
```

Comment:
Seems that is not actually a maximum because you don't have an explicit range. e.g. minimum = 0.1 maximum = 1.0
The behavior for this property is to specify the width ratio.

Reply:
As above I mentioned, this value is not for the actual width ratio, but only limiting the maximum width of scale bar(the actual width is dynamic and depends on zoom level)